src/pages/HomePage.tsx

```diff
@@ -60,10 +60,13 @@ function HomePage() {
             software, design, and strategy to complete awesome projects.
           </p>
           <p className="regularText">
-            At Tech Start, our members learn new technologies, gain experience
-            with professional development tools and methodologies, and master
-            the art of collaboration. We're also the top spot on campus to meet
-            fellow tech enthusiasts, aspiring entrepreneurs, and innovators.
+            At Tech Start, our members learn to use new technologies, gain
+            experience with professional development tools and methodologies,
+            and master the art of collaboration. You do not need to know the
+            skills before hand! Here at Tech Start, we teach all necessary
+            skills required for you to succeed here and in industry. We're also
```

Comment:
Maybe abbreviate this to "We give our members the support and mentorship they need to succeed regardless of their previous experience"

Reply:
Thank you for the feedback! Made the changes just now.